fs/shared/src/main/scala/rapture/fs/files.scala

```diff
@@ -127,10 +127,10 @@ trait LowPriorityImplicits extends LowerPriorityImplicits {
 
   /** Specifies how file: URLs should be navigable. */
   implicit object NavigableFile extends Navigable[FsUrl] {
-    def children(url: FsUrl)(implicit mode: Mode[`Navigable#children`]): mode.Wrap[List[FsUrl], Exception] =
+    override def children(url: FsUrl)(implicit mode: Mode[`Navigable#children`]): mode.Wrap[Seq[FsUrl], Exception] =
```

Comment:
Oh... I notice you've added `override`. Was the previous implementation just *shadowing* the inherited version before?